src/KDSoapClient/KDSoapClientInterface.cpp

```diff
@@ -58,7 +58,7 @@ void KDSoapClientInterface::setSoapVersion(KDSoapClientInterface::SoapVersion ve
     d->m_version = static_cast<KDSoap::SoapVersion>(version);
 }
 
-KDSoapClientInterface::SoapVersion KDSoapClientInterface::soapVersion()
+KDSoapClientInterface::SoapVersion KDSoapClientInterface::soapVersion() const
```

Comment:
This is unrelated, and I'm pretty sure I saw it in a different review request already, please remove from this commit.